docs/products/clickhouse/howto/configure-tiered-storage.rst

```diff
@@ -0,0 +1,94 @@
+
+You may want to change this default data distribution behavior by :ref:`configuring your table's schema by adding a TTL (time-to-live) clause <time-based-retention-config>`. Such a configuration allows ignoring the SSD-capacity threshold and moving the data from SSD to the object storage based on how long the data is there on your SSD.
+
+To enable this time-based data distribution mechanism, you can set up a retention policy (threshold) on a table level by using the TTL clause. For data retention control purposes, the TTL clause uses the following:
+
+* Data item of the `Date` or `DateTime` type as a reference point in time
+* INTERVAL clause as a time period to elapse between the reference point and the data transfer to the object storage
+
+Prerequisites
+-------------
+
+* Aiven organization
+* Tiered storage feature :doc:`enabled </docs/products/clickhouse/howto/enable-tiered-storage>` on the project level and on the table level
+* Command line tool (:doc:`ClickHouse client </docs/products/clickhouse/howto/connect-with-clickhouse-cli>`)
+
+.. _time-based-retention-config:
+
+Configure time-based data retention
+-----------------------------------
+
+1. :doc:`Connect to your Aiven for ClickHouse service </docs/products/clickhouse/howto/list-connect-to-service>` using, for example, the ClickHouse client (CLI).
+2. Select a database for operations you intend to perform.
+
+   .. code-block:: bash
+
+      USE database-name
+
+Add TTL to a new table
+''''''''''''''''''''''
+
+Create a new table with the ``storage_policy`` setting set to ``tiered`` (to :doc:`enabled </docs/products/clickhouse/howto/enable-tiered-storage>` the feature) and TTL (time-to-live) configured to add a time-based data retention threshold on the table.
+
+.. code-block:: shell
+
+    CREATE TABLE example_table (
+        SearchDate Date,
+        SearchID UInt64,
+        SearchPhrase String
+    )
+    ENGINE = MergeTree
+    ORDER BY (SearchDate, SearchID)
+    PARTITION BY toYYYYMM(SearchDate)
+    TTL SearchDate + INTERVAL 1 WEEK TO VOLUME 'tiered'
+    SETTINGS storage_policy = 'tiered';
+
+Add TTL to an existing table
+''''''''''''''''''''''''''''
+
+Use the MODIFY TTL clause:
+
+.. code-block:: shell
+
+    ALTER TABLE database_name.table_name MODIFY TTL ttl_expression;
+
+Update TTL to an existing table
+'''''''''''''''''''''''''''''''
+
+Change an already configured TTL in an existing table by using the ALTER TABLE MODIFY TTL clause:
+
+.. code-block:: shell
+
+    ALTER TABLE database_name.table_name MODIFY TTL ttl_expression; 
+
+.. topic:: Result
+   
+   You have your time-based data retention policy set up. Now when a piece of data is there on your SSD for a specified time period, it's moved to the object storage, regardless of how much of the SSD capacity is still available.
```

Comment:
"Now when data has been on your SSD for longer than the specified time period, it is moved to object storage, regardless of how much SSD capacity is still available."

Reply:
has been - giving preference to simple present (zero conditional)
for longer - removing "for"
the specified time period - changing "the" to "a" (it's not one generic period)
Applying this comment partially.